google/cloud/bigtable/client.py

```diff
@@ -212,23 +212,19 @@ def _create_gapic_client_channel(self, client_class, grpc_transport):
         if self._client_options and self._client_options.api_endpoint:
             api_endpoint = self._client_options.api_endpoint
         else:
-            api_endpoint = client_class.SERVICE_ADDRESS
+            api_endpoint = client_class.DEFAULT_ENDPOINT
 
         channel = grpc_transport.create_channel(
-            api_endpoint,
-            self._credentials,
+            host=api_endpoint,
+            credentials=self._credentials,
             options={
                 "grpc.max_send_message_length": -1,
                 "grpc.max_receive_message_length": -1,
                 "grpc.keepalive_time_ms": 30000,
                 "grpc.keepalive_timeout_ms": 10000,
             }.items(),
         )
-        transport = grpc_transport(
-            address=api_endpoint,
-            channel=channel,
-            credentials=None,
```

Comment:
@crwilcox confirming removing `credentials` here is ok? (looked at firestore)

Reply:
should be the same. :)